network-api/networkapi/wagtailcustomization/redirects/middleware.py

```diff
@@ -11,7 +11,9 @@ def _new_get_redirect(request, path):
         locale_prefix = f'/{request.LANGUAGE_CODE}/'
         if path.startswith(locale_prefix):
             path = path.replace(locale_prefix, '/', 1)
-
+    if 'QUERY_STRING' in request.META:
```

Comment:
checking if there are any utm params in the request, if so, append them to the updated path so they do not get removed